### DISCUSSION TITLE
romeo's multiple ideas and suggestions

### DISCUSSION THREAD

**Discussion by romeogiggletoes** [Chatterino/chatterino2#3453] [Ideas]
Things I'd like added/changed are: 
~~1. Custom value zooms for things like interface zoom and emote size (not just limited preset values to choose from)~~ _Already supported_

~~2. Change 'large' option for link previews to be bigger (alternatively make an extra large option) My display might just be bugged though I can send a screen shot of it: https://i.imgur.com/bwt7S2l.png~~ _Opened a separate issue on this, see #3462_

3. Ability to disable Event badges separately instead of Vanity badges like ffz does (i.e. GlitchCon, TwitchCon, etc.)
_see #3461 for full suggestion_

~~4. Ability to limit the amount of chat history not just for startup since eventually it'll just go back to that 800 (this is probably the most important out of this list since I like having a lot of channels open and this would free up a lot of ram)~~ _Already an open issue, see below_

~~5. Ability to disable emote padding https://i.imgur.com/s02KeDq.png~~ _Impossible to accomplish realistically_

6. Ability to change the amount of space between two different messages 
![2022_01_03_16_49_23_Chatterino_2 3 4](https://user-images.githubusercontent.com/97059214/147984330-c3377372-3a35-4f10-a272-afbf950fa469.png) 
_see #3460 for full suggestion_

~~7. Add a highlight color for whispers but it seems like there's no box for it https://i.imgur.com/dzmBEQg.png~~ _Already an open issue, see below_

8. Ability to change the size and position of the emote autocomplete menu. See #2389 comments

Edit: Cleaned up wording and updated the ideas.

first post on github pajaW

**Comment by Mm2PL** [comment]
1. custom value zooms for things like interface zoom and emote size (not just limited preset values to choose from)

This is a UI/UX issue. You can type your custom zoom level in the box.

2. would like 'large' option for link previews to be bigger (alternatively make an extra large option) My display might just be bugged though I can send a screen shot of it: i.imgur.com/bwt7S2l.png

Yeah they can be small or too small making things hard to read on them for example. Agree with you on this one hard.

3. would like to be able to disable more 'vanity' badges separately like ffz does (i.e. GlitchCon, TwitchCon, etc.)

I'd like for this to get delayed until we actually rework the settings dialog.

4. I'd like to be able to limit the amount of chat history not just for startup since eventually it'll just go back to that 800 (this is probably the most important out of this list since I like having a lot of channels open and this would free up a lot of ram)

There is an issue for this https://github.com/Chatterino/chatterino2/issues/11. The setting you are changing is for loading history at start up.

5. a setting to disable emote padding
![2022-01-03_22-17](https://user-images.githubusercontent.com/25011746/147981351-2e25e625-bc50-4297-97c5-e94d36f73494.png)
I think this is the setting you're looking for.

6. a setting to change the amount of space between two different messages

We don't have such a thing, but there is technically code that does something similar. Note to self: message element? flag for this

7. Also I'd like to add a highlight color for whispers but it seems like there's no box for it i.imgur.com/dzmBEQg.png

Found this issue: https://github.com/Chatterino/chatterino2/issues/1922

**Reply by romeogiggletoes** [reply]
1. I am dumb
2. Will make separate issue post about this
3. 👍 
4. I'm guessing then that it's just waiting to get resolved?
5. Sadly it is not, I have uploaded an image to show what I am referring to. To further elaborate, I would like a setting for emote padding disabled and so the emote will just overlap the text that's above it.
6. 👍  (will also upload image of what I'm referring to for more clarity)
7. Same thing as 4th suggestion I'm guessing

Edit: I have already made separate issued posts about suggestions that didn't already have an issue before.

**Reply by romeogiggletoes** [reply]
well would you look at that #2722 @Mm2PL
